collector/container/konflux.Dockerfile

```diff
@@ -160,9 +192,6 @@ LABEL \
 
 WORKDIR /
 
-COPY collector/container/rhel/install.sh /
```

Comment:
I replaced this with https://github.com/stackrox/collector/pull/1573/files#diff-dd2e1150313169ab18110018f7c1e085ab8c31e164916b63883f5962afb49e79R155-R165